### PR TITLE
Corrected code to close secure storage dialog.

### PR DESCRIPTION
**Pull request by ldimaggi**
Cleaned up the code used to close the secure storage window.

### CODE REVIEW

**Comment by vpakan** [comment]
* do not pass log to TestSupport.java methods define Logger instance in TestSupport.java instead the same way as Logger is defined within other classes as final private static
* do not use capital letter for variable name https://github.com/jbosstools/jbosstools-integration-tests/pull/1033/files#diff-dee7ad977ad5faeda09371034fadeed8R114
* do not use try catch blog in https://github.com/jbosstools/jbosstools-integration-tests/pull/1033/files#diff-dee7ad977ad5faeda09371034fadeed8R97 this should be working always I guess your code is not just trying if it's possible
* do not forget to create similar PR also for jbosstools-4.3.-.Beta1x branch

File: plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java [hunk 1]
```diff
@@ -12,11 +16,17 @@
 import org.jboss.reddeer.swt.impl.shell.DefaultShell;
 import org.jboss.reddeer.swt.impl.tree.DefaultTree;
 import org.jboss.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.jboss.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
 
 public class TestSupport {
 	
+	/* Cannot reference "this" in a static context - the following cannot be used here:
```
Comment (vpakan) [inline plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java]: Please remove this comment it's obvious

File: plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java [hunk 2]
```diff
@@ -12,11 +16,17 @@
 import org.jboss.reddeer.swt.impl.shell.DefaultShell;
 import org.jboss.reddeer.swt.impl.tree.DefaultTree;
 import org.jboss.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.jboss.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
 
 public class TestSupport {
```
Comment (vpakan) [inline plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java]: Please remove this comment it's obvious

File: plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java [hunk 3]
```diff
@@ -12,11 +16,17 @@
...
 	
+	/* Cannot reference "this" in a static context - the following cannot be used here:
+	 * Logger log = Logger.getLogger(this.getClass());
+	 */
+	private static final Logger log = Logger.getLogger(TestSupport.class);
+	
 	/* Test Setup part 1 */
-	public static List<TreeItem> mylynTestSetup1 (Logger log) {		
+	public static List<TreeItem> mylynTestSetup1 () {
```
Comment (vpakan) [inline plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java]: Please do not use prefix like this "*** step" use log.debug() or log.trace() and message without prefix instead

**Comment by vpakan** [comment]
Please create similar PR also for jbosstools-4.3.0.Beta1x branch and rebase it properly

**Comment by nickboldt** [comment]
PR for jbosstools-4.3.0.Beta1x, cherry-picking only a2aede7 - #1034

**Pull request closed without merging**